lib/frameworks/debugprint.ts

```diff
@@ -11,11 +12,9 @@ var logger = new Logger('debugger');
  * @param {Array} specs Array of Directory Path Strings.
  * @return {Promise} Promise resolved with the test results
  */
-exports.run = (runner, specs) => {
+export const run = (runner: Runner, specs: Array<string>) => {
   return new Promise(resolve => {
-    logger.info('Resolved spec files: ' + util.inspect(specs));
-    resolve({
-      failedCount: 0
-    });
+    logger.info(`Resolved spec files: ${util.inspect(specs)}`);
+    resolve({failedCount: 0});
```

Comment:
Once the changes are made to `RunResults` you probably do not need to cast. If for some reason there are type errors, you might need to cast this to `resolve({failedCount: 0} as RunResults);`